.gitignore

```diff
@@ -27,6 +27,7 @@ dist/
 node_modules/
 local/
 ui/*/npm-debug.log
+ui/*/yarn.lock
```

Comment:
yarn lock files need to be checked into git. This allows the process to compare against previous version. If no dependency in package.json has changed "yarn install" results are pulled from previous run.

https://yarnpkg.com/lang/en/docs/yarn-lock/#toc-check-into-source-control

Once you have an update for any dependency in package.json - yarn.lock file should also be updated in git.